<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentPage</class>
 <widget class="QWidget" name="PaymentPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>860</width>
    <height>560</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #23272A;</string>
  </property>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>820</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 900 18pt &quot;Segoe UI&quot;; color: white;</string>
   </property>
   <property name="text">
    <string>Payment</string>
   </property>
  </widget>
  <widget class="QLabel" name="reservationInfoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>820</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;; color: white;</string>
   </property>
   <property name="text">
    <string>Reservation: Hall A - Jan 1, 2023 - 2 hours</string>
   </property>
  </widget>
  <widget class="QLabel" name="amountLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;; color: white;</string>
   </property>
   <property name="text">
    <string>Amount to Pay:</string>
   </property>
  </widget>
  <widget class="QLabel" name="amountValueLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Segoe UI&quot;; color: #57F287;</string>
   </property>
   <property name="text">
    <string>$200.00</string>
   </property>
  </widget>
  <widget class="QLabel" name="paymentMethodLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;; color: white;</string>
   </property>
   <property name="text">
    <string>Payment Method:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="paymentMethodComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QComboBox {
       background-color: #1E1F22;
       color: white;
       border-radius: 8px;
       padding-left: 10px;
     }
     QComboBox:hover {
       border: 1px solid #5865F2;
     }
    </string>
   </property>
   <item>
    <property name="text">
     <string>Credit Card</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Debit Card</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PayPal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Bank Transfer</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="cardNumberLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;; color: white;</string>
   </property>
   <property name="text">
    <string>Card Number</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardNumberLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>300</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QLineEdit {
       background-color: #1E1F22;
       color: white;
       border-radius: 8px;
       padding-left: 10px;
     }
     QLineEdit:focus {
       border: 1px solid #5865F2;
     }
    </string>
   </property>
   <property name="placeholderText">
    <string>1234 5678 9012 3456</string>
   </property>
  </widget>
  <widget class="QLabel" name="expiryLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;; color: white;</string>
   </property>
   <property name="text">
    <string>Expiry Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="expiryLineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>340</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QLineEdit {
       background-color: #1E1F22;
       color: white;
       border-radius: 8px;
       padding-left: 10px;
     }
     QLineEdit:focus {
       border: 1px solid #5865F2;
     }
    </string>
   </property>
   <property name="placeholderText">
    <string>MM/YY</string>
   </property>
  </widget>
  <widget class="QLabel" name="cvvLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>310</y>
     <width>200</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 12pt &quot;Segoe UI&quot;; color: white;</string>
   </property>
   <property name="text">
    <string>CVV</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cvvLineEdit">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>140</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QLineEdit {
       background-color: #1E1F22;
       color: white;
       border-radius: 8px;
       padding-left: 10px;
     }
     QLineEdit:focus {
       border: 1px solid #5865F2;
     }
    </string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::EchoMode::Password</enum>
   </property>
   <property name="placeholderText">
    <string>123</string>
   </property>
  </widget>
  <widget class="QPushButton" name="payButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QPushButton {
       font: 700 14pt &quot;Segoe UI&quot;;
       color: white;
       background-color: #57F287;
       border-radius: 12px;
     }
     QPushButton:hover {
       background-color: #3BA55C;
     }
    </string>
   </property>
   <property name="text">
    <string>Pay Now</string>
   </property>
  </widget>
  <widget class="QFrame" name="confirmationFrame">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>480</width>
     <height>380</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QFrame {
       background-color: #1E1F22;
       border-radius: 15px;
     }
    </string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="confirmationIconLabel">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 72 48pt &quot;Segoe UI&quot;; color: #57F287;</string>
    </property>
    <property name="text">
     <string>âœ“</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="confirmationLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 16pt &quot;Segoe UI&quot;; color: white;</string>
    </property>
    <property name="text">
     <string>Payment Successful!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="confirmationDetailsLabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>380</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Segoe UI&quot;; color: white;</string>
    </property>
    <property name="text">
     <string>Transaction ID: CHR-123456789
Amount: $200.00
Date: Jan 1, 2023</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="doneButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>180</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
      QPushButton {
        font: 700 12pt &quot;Segoe UI&quot;;
        color: white;
        background-color: #5865F2;
        border-radius: 12px;
      }
      QPushButton:hover {
        background-color: #4752C4;
      }
     </string>
    </property>
    <property name="text">
     <string>Done</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>460</y>
     <width>300</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
     QPushButton {
       font: 700 14pt &quot;Segoe UI&quot;;
       color: white;
       background-color: rgb(255, 0, 0);
       border-radius: 12px;
     }
     QPushButton:hover {
       background-color: rgb(170, 0, 0);
     }
    </string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
